src/api/FamilyAPI.ts

```diff
@@ -20,8 +20,8 @@ const getFamiliesByParentName = (
 const getFamilyById = (id: number): Promise<FamilyDetailResponse> =>
   APIUtils.get(`/families/${id}`);
 
-const postFamily = (data: FamilyStudentRequest) =>
-  APIUtils.post("/families/", data);
+const postFamily = (data: FamilySessionRequest) =>
+  APIUtils.post("/enrolments/", data);
```

Comment:
can we remove postFamily here (since we're no longer using it) and create a new EnrolmentAPI.ts file for this?